PowerShell.Common.props

```diff
@@ -63,7 +63,8 @@
       <PackageVersion>$(PSCoreBuildVersion)</PackageVersion>
 
       <ApplicationIcon Condition="$(ProductVersion.Contains('preview'))">..\..\assets\Powershell_av_colors.ico</ApplicationIcon>
-      <ApplicationIcon Condition="!$(ProductVersion.Contains('preview'))">..\..\assets\Powershell_black.ico</ApplicationIcon>
+      <ApplicationIcon Condition="$(ProductVersion.Contains('daily'))">..\..\assets\Powershell_avatar.ico</ApplicationIcon>
```

Comment:
can't wait for the day where we can use Choose/When/Otherwise in Targets.............

https://github.com/Microsoft/msbuild/issues/2056

Reply:
Easier to rewrite MSBuild in PowerShell 😄